client/src/components/InputField/InputField.js

```diff
@@ -55,7 +77,58 @@ class InputField extends Component {
     }
   }
 
+  handleTipToggle() {
+    this.setState((state) => ({ tipOpen: !state.tipOpen }));
+  }
+
+  renderTip() {
+    const id = this.props.id || this.props.name;
+    const title = this.props.title;
+    const { icon = 'lamp', importance = 'normal', content } = this.props.tip;
+    const { tipOpen } = this.state;
+
+    const tipIconColor = tipImportanceColorMap[importance];
+    const tipType = tipImportanceMessageMap[importance];
```

Comment:
Feels like those values should be customisable.

Reply:
You can customise the icon, but the colour and description are baked into the importance levels for consistency. @clarkepaul - any thoughts on whether the icon colour and accessible description of the button should be directly modifiable?

Reply:
I think we should only allow colours that we recommend for select purposes. If they are completely customisable you'll get people using all sorts of random colours which won't have much meaning for the end-user and will contradict the other system colours. The colours we provide should align with system alerts, badges... that sort of thing. Happy to add new colours as they are needed, so trying to keep them as few as possible to start with.